mlblocks_primitives/mlprimitives.timeseries_errors.extract_anomalies.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "mlprimitives.timeseries_errors.extract_anomalies",
+    "author": "Ihssan Tinawi <itinawi@mit.edu>",
+    "description": "mlprimitives.timeseries_errors.extract_anomalies",
+    "classifiers": {
+        "type": "preprocessor",
+        "subtype": "feature_extractor"
+    },
+    "modalities": ["timeseries"],
+    "primitive": "mlprimitives.timeseries_errors.extract_anomalies",
+    "produce": {
+        "args": [
+            {
+                "name": "y_true",
+                "type": "np.array"
+            },
+            {
+                "name": "smoothed_errors",
+                "type": "list"
+            },
+            {
+                "name": "window_size",
```

Comment:
Please remove any hyperparameters from here. This list should contain only the function arguments that are required during fit and predict, not the hyperparameters.